packages/nimble-components/build/generate-workers/source/matrix-renderer.ts

```diff
@@ -8,16 +9,158 @@ import { expose } from 'comlink';
  * This setup is used in the wafer-map component to perform heavy computational duties
  */
 export class MatrixRenderer {
-    public dieMatrix: Uint8Array = Uint8Array.from([]);
+    public colIndexes: Int32Array = Int32Array.from([]);
+    public rowIndexes: Int32Array = Int32Array.from([]);
+    public values = new Float64Array([]);
+    public scaledColIndex = new Float64Array([]);
+    public scaledRowIndex = new Float64Array([]);
+    public colIndexPositions: Int32Array = Int32Array.from([]);
+    public canvas!: OffscreenCanvas;
+    public context!: OffscreenCanvasRenderingContext2D;
+    private scaleX: number = 1;
+    private scaleY: number = 1;
+    private baseX: number = 1;
+    private baseY: number = 1;
+    private dieDimensions: Dimensions = { width: 1, height: 1 };
+    private transform: Transform = { k: 1, x: 0, y: 0 };
+    private topLeftCanvasCorner!: { x: number; y: number; };
+    private bottomRightCanvasCorner!: { x: number; y: number; };
+    private margin: { top: number; right: number; bottom: number; left: number; } = { top: 20, right: 20, bottom: 20, left: 20 };
+
+    public setColIndexes(colIndexesBuffer: Int32Array): void {
+        this.colIndexes = colIndexesBuffer;
+        const scaledColIndex = [this.scaleX * this.colIndexes[0]! + this.baseX + this.margin.left];
+        const colPosition = [0];
```

Comment:
`columnPositions`

Reply:
renamed it!
![image](https://github.com/ni/nimble/assets/110180309/e5fe45cf-4cdb-4817-a895-7bf5a17677cf)

Reply:
![image](https://github.com/ni/nimble/assets/110180309/7251047b-c94b-4a56-b6f9-42b25e6dd949)
renamed it!